python/lsst/daf/butler/cli/butler.py

```diff
@@ -0,0 +1,174 @@
+# This file is part of daf_butler.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import click
+import logging
+import os
+
+import lsst.log
+from lsst.utils import doImport
+
+
+def _initLogging(logLevel):
+    log = lsst.log.Log.getLogger("lsst.daf.butler")
+    if "critical" == logLevel:
```

Comment:
I thought I made a comment on this but I can't find it any more. Just in case it got lost: please don't do this with `if` statements. The logging module has methods for translating the text string to a logging level and lsst.log has method for converting a logging level to an lsst.log level. See astro_metadata_translator command line tool for example code.